.gitignore

```diff
@@ -54,8 +54,5 @@ etc/tmp/*
 ### Database files ###
 db/
 *.db
-openehr/ckm/ckm.sqlite
 
 ### openehr ckm files ###
```

Comment:
As the ckm files are removed from this ignore, this comment line should be removed too.